mchf-eclipse/.settings/org.eclipse.cdt.managedbuilder.core.prefs

```diff
@@ -0,0 +1,13 @@
+eclipse.preferences.version=1
```

Comment:
I think we decided not to share this file. Can't remember the reason, to be honest. Might be that every Eclipse install will change this file and that causes lots of changes for nothing.

Reply:
I suppose that this is by doing "git add . "

Reply:
git add . is an absolute no-go! I use git-gui (installed as part of Windows git) to control what I commit (I don't use the Eclipse integrated git support). This is much more convenient than the command line for that purpose. Most other stuff I use the command line but for preparing and managing the content of a commit git-gui is much better than the command line.